buildsystem/codecompliance/util.py

```diff
@@ -68,3 +88,14 @@ def findfiles(paths, exts):
 
             if has_ext(filename, exts):
                 yield filename
+
+
+def issue_str(title, filename):
+    return (title, filename)
```

Comment:
This function from which all (should/will) issues pass, so if we want to add other format or color we can just change this only function. You don't agree?